commit 6c3117510493de8c18e1b6d936de8831c1b2918e (refs/changes/03/4403/2)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2018-06-21T19:39:27+00:00 (1 year, 4 months ago)
    
    TRITON-535 Need to update sdc-sdc's sdc-napi-ufds-watcher submodule
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/deps/sdc-napi-ufds-watcher b/deps/sdc-napi-ufds-watcher
index 1bc4077..2d4e757 160000
--- a/deps/sdc-napi-ufds-watcher
+++ b/deps/sdc-napi-ufds-watcher
@@ -1 +1 @@
-Subproject commit 1bc40777f03ae3a2d6e5eabc2b441c9c7cd8404d
+Subproject commit 2d4e75711d2212d808bb167b0b3c8d9eecb7d7f1
